Calendar/Get-RBASummary.ps1

```diff
@@ -266,7 +280,10 @@ function InPolicyProcessing {
         Write-Host "`t BookInPolicy:                     {$($RbaSettings.BookInPolicy)}"
     } else {
         Write-Host "`t BookInPolicy:                     These $($RbaSettings.BookInPolicy.count) accounts do not require the delegate approval."
-        foreach ($BIPUser in $RbaSettings.BookInPolicy) { Write-Host " `t `t $BIPUser " }
+        foreach ($BIPUser in $RbaSettings.BookInPolicy) {
+            $BIPUserMB = Get-Mailbox -Identity $BIPUser
+            Write-Host " `t `t [$($BIPUserMB.DisplayName)] -- $($BIPUserMB.PrimarySmtpAddress)"
+        }
```

Comment:
You are doing this same loop over and over again. Just a different list. I would make this info a function that you pass a list of loop over that and have it do everything you want.

Reply:
Done